src/Shared/AppModels/AppRoot.cs

```diff
@@ -245,6 +204,7 @@ public async Task InitAsync(CancellationToken cancellationToken = default)
                 return;
             }
 
+            Guard.IsNotNull(Ipfs);
```

Comment:
This line needs to be removed. Ipfs is optional in Strix, this will crash if the user doesn't have IPFS enabled.